wpi-many.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
+
+# This script runs the Checker Framework's whole-program inference on each of a list of projects.
+
+### Usage
+#
+# See the README.md file that accompanies this script.
+#
+
+### Dependencies:
+#
+# - JAVA8_HOME environment variable must point to a Java 8 JDK
+# - JAVA11_HOME environment variable must point to a Java 11 JDK
+# - CHECKERFRAMEWORK environment variable must point to a built copy of the Checker Framework
+# - Other dependencies: perl, python2.7 (for dljc), awk, git, mvn, gradle, wget, curl
+#
+
+### Required arguments:
+#
+# -o outdir : run the experiment in the ./outdir directory, and place the
+#             results in the ./outdir-results directory. Both will be created
+#             if they do not exist.
+#
+# -i infile : read the list of repositories to use from the file $infile. Each
+#             line should contain the (https) url of the git repository on
+#             GitHub and the commit hash to use, separated by whitespace.
+#             Repositories must be of the form:
+#             https://github.com/username/repository - the script is reliant
+#             on the number of slashes, so excluding https:// is an error.
+#             
+#             If the repository's owner is the user specified by the -u flag,
+#             then each line owned by that user must contain a third element,
+#             the original github repository.
+#
+
+### Optional arguments:
+#
+# -u user : the GitHub owner for repositories that have
+#           been forked and modified. These repositories must have a third entry
+#           in the infile indicating their origin. Default is "$USER".
+#
+# -t timeout : the timeout for running the checker (via WPI) on a single project, in seconds
+#
+# Any arguments that follow these arguments (and are separated by a
+# literal "--") are passed directly to DLJC without modification. See
+# the documentation of DLJC for an explanation of these arguments:
+# https://github.com/kelloggm/do-like-javac . At least one such argument
```

Comment:
I don't see documentation of the arguments there.  In particular, I looked for documentation of `--stubs` and did not find it.

Reply:
The documentation is available by running `python2 dljc --help`, which includes [this code](https://github.com/kelloggm/do-like-javac/blob/master/do_like_javac/arg.py#L57). All of dljc's options are documented this way, but I agree that it might be easier for users if the README included that information. I'll change this comment to specify that it is necessary to run `python2 dljc --help`